general/documentation/installation.md

```diff
@@ -58,19 +58,23 @@ Gitpod is an alternative to local development and completely optional. We recomm
 
 ## Useful commands
 
-### Starting the development server
+### Starting the development server - the best way to edit docs locally
```

Comment:
Rather than having a reaaaaallly long title, can I suggest putting this as prose.
This long title will impact readability in the sidebar nav.

```suggestion
### Starting the development server

The development server is the best way to edit docs locally. Once the development server has loaded you can make changes to the source markdown files (`*.md` and `*.mdx`), which should be automatically live reloaded in your browser.
```